stubs/mysqlclient/MySQLdb/cursors.pyi

```diff
@@ -34,26 +36,29 @@ class BaseCursor:
     def setinputsizes(self, *args) -> None: ...
     def setoutputsizes(self, *args) -> None: ...
     def execute(self, query, args: Incomplete | None = None): ...
-    def executemany(self, query: str, args: list[Any]) -> int: ...
+    def mogrify(
+        self, query: str | bytes, args: dict[str, _Literal] | dict[bytes, _Literal] | Iterable[_Literal] | None = None
```

Comment:
Does `dict[str, _Literal]` work despite the invariance of `dict`? `_Literal` contains `Any`, but also other types. Should we use `dict[str, Any]`, instead?

Reply:
I think what you've got _should_ be fine. If it causes issues for somebody, we'll probably get a complaint about it, and we can change it then ;)